runners/flink/runner/src/test/java/org/apache/beam/runners/flink/streaming/TestCountingSource.java

```diff
@@ -0,0 +1,256 @@
+/*
```

Comment:
This is copied from "org.apache.beam.sdk.runners.dataflow.TestCountingSource". I wonder if there is a way to avoid duplication by moving this class.